pkg/client/cluster-essentials/clusterinit.go

```diff
@@ -0,0 +1,81 @@
+// Copyright 2023 VMware, Inc. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+// Package clusteressentials has APIs for bootstrapping kubernetes cluster
+package clusteressentials
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"time"
+
+	"k8s.io/client-go/rest"
+
+	"github.com/vmware-tanzu/tanzu-framework/pkg/client/cluster-essentials/pkg/utils"
+)
+
+type ClusterEssentialsClient struct {
+	ClusterEssentialBundlePath string
+	BundleDestDir              string
```

Comment:
What is the advantage in exposing this field? It's just unpacking the bundle in a (temp) directory.